README.md

```diff
@@ -397,14 +398,124 @@ constructor(props) {
 
 # Lists
 
-> TODO
-> ng-repeat, trackby vs key
+### AngularJS
+```js
+export class BookListComponentCtrl {
+  constructor() {
+    this.books = [
+      {
+        id: 1,
+        title: "Eloquent JavaScript",
+        author: "Marijn Haverbeke"
+      },
+      {
+        id: 2,
+        title: "JavaScript: The Good Parts",
+        author: "Douglas Crockford"
+      },
+      {
+        id: 3,
+        title: "JavaScript: The Definitive Guide",
+        author: "David Flanagan"
+      }
+    ];
+  }
+}
+```
+
+```html
+<ul>
+  <li ng-repeat="book in $ctrl.books track by book.id">
+    {{ book.title }} by {{ book.author }}
+  </li>
+</ul>
+```
+
+### Angular
+```js
+@Component({
+  selector: 'book-list',
+  template: `
+    <ul>
+      <li *ngFor="let book of books; trackBy: trackById">
+        {{ book.title }} by {{ book.author }}
+      </li>
+    </ul>
+  `
+})
+export class BookListComponent {
+  this.books = [
+    {
+      id: 1,
+      title: "Eloquent JavaScript",
+      author: "Marijn Haverbeke"
+    },
+    {
+      id: 2,
+      title: "JavaScript: The Good Parts",
+      author: "Douglas Crockford"
+    },
+    {
+      id: 3,
+      title: "JavaScript: The Definitive Guide",
+      author: "David Flanagan"
+    }
+  ];
```

Comment:
Just add type definition as it's in TypeScript;

Reply:
Added interface example.